enterprise/cli/provisionerdaemons_test.go

```diff
@@ -36,6 +36,32 @@ func TestProvisionerDaemon_PSK(t *testing.T) {
 	pty.ExpectMatchContext(ctx, "starting provisioner daemon")
 }
 
+func TestProvisionerDaemon_Named(t *testing.T) {
+	t.Parallel()
+
+	client, _ := coderdenttest.New(t, &coderdenttest.Options{
+		ProvisionerDaemonPSK: "provisionersftw",
+		LicenseOptions: &coderdenttest.LicenseOptions{
+			Features: license.Features{
+				codersdk.FeatureExternalProvisionerDaemons: 1,
+			},
+		},
+	})
+
+	inv, conf := newCLI(t, "provisionerd", "start",
+		"--psk=provisionersftw",
+		"--name=matt-daemon",
```

Comment:
As we have this test, maybe we should drop `TestProvisionerDaemon_PSK` as they have similar coverage.

Reply:
Fair, or I can just piggy-back on _PSK.